### ISSUE TITLE
Automatically select image on tap

### ISSUE THREAD

**Issue by surdu** [NativeScript/nativescript-imagepicker#163]
Now, in iOS, even if you are using `mode: "single"`, you have to select the image from the gallery and then press the `Done` button.

The UX might be improved here slightly by considering the tapped image as selected when in single mode.

Maybe this will not be the case anymore after #156 will be fixed, but just wanted to make sure.

**Comment by radeva** [comment]
Thanks @surdu for reporting this UX improvement. We will definitely take it into account when we start implementing #156.

**Comment by tbozhikov** [comment]
This is already done in [6.0.0](https://github.com/NativeScript/nativescript-imagepicker/releases/tag/v6.0.0), I am closing the issue.
